debugoverlay-ext-netstats/src/main/java/com/ms_square/debugoverlay_ext_netstats/NetStatsDataModule.java

```diff
@@ -0,0 +1,76 @@
+package com.ms_square.debugoverlay_ext_netstats;
+
+import android.net.TrafficStats;
+import android.os.Handler;
+import android.os.Looper;
+
+import com.ms_square.debugoverlay.modules.BaseDataModule;
+
+import java.util.Locale;
+
+public class NetStatsDataModule extends BaseDataModule<String> {
+
+    private static final String TAG = NetStatsDataModule.class.getSimpleName();
+
+    private static final String HEADER = "Received: %8s/s\nSent: %12s/s";
+
+    private Handler handler = new Handler(Looper.getMainLooper());
+
+    private final int interval;
+    private final int uid;
+
+    private double mPreviousReceived = 0;
```

Comment:
please avoid using m prefix to be consistent with the rest of the code. Also initializing it to 0 is redundant here since java's initializer will initialize it to 0.